charts/schema-registry/values.yaml

```diff
@@ -67,8 +67,10 @@ authentication:
   ## authentication is enabled
   passwordAuthSecret:
   userPasswordFile:
-  authorizationResourceQualifier: ""
   disableBasicAuthentication: false
+  authResourceDomain: "prn"
```

Comment:
I suppose this makes these values configurable at deployment time. Do you really need users to configure them? It'd make sense to do that if the idea is to keep them product agnostic. 

```
authResourceDomain
authNamespaceResourceIdentifier
authGroupResourceIdentifier
```

Reply:
yeah, the idea is to allow admins to create single set of ACLs for multiple related resoures across pravega and schema registry for example.
Typical usage of schema registry in pravega context would be to use namespace as analogous to scopes and groups as analogous to streams.